pkg/kubectl/cmd/drain.go

```diff
@@ -495,8 +495,10 @@ func (o *DrainOptions) evictPods(pods []api.Pod, policyGroupVersion string, getP
 				if err == nil {
 					break
 				} else if apierrors.IsTooManyRequests(err) {
-					time.Sleep(5 * time.Second)
-				} else {
+					retryInterval := 5
+					fmt.Fprintf(o.ErrOut, "Cannot evict pod as it would violate the pod's disruption budget. Retry in %v seconds\n", retryInterval)
```

Comment:
We would generally always prefix a message to stderr with `error: `, `warning: `, or `info: `.  In this case, it's `warning: `.

Try to make it a single sentence - `warning: Evicting pod would violate disruption budget, retrying in %d seconds ...`